FOUNDATIONS.md

```diff
@@ -13,15 +13,12 @@ ensure the success of future improvements (especially type systems).
 
 ## Linked
 
-IPLD must support linking to any IPLD node (even if the node is in the middle of
-a block). That is, IPLD must support arbitrary IPLD paths in links.
+The IPLD Data Model includes Links. A Link can be resolved to reach another IPLD Node.
```

Comment:
"another" seems misplaced here. Will "an" suffice?

Reply:
That reads weirder to me.  It's definitely not reaching the same node we started on.